config/initializers/sync-database.js

```diff
@@ -5,10 +5,10 @@ export default {
   after: 'define-orm-models',
   async initialize(application) {
     let container = application.container;
-    let adapter = container.lookup('orm-adapter:node-orm2');
+    let db = container.lookup('database:orm2', { loose: true });
 
-    if (application.config.database.syncSchema) {
-      await fromNode((cb) => adapter.db.sync(cb));
+    if (db && application.config.database.sync) {
```

Comment:
Why is a null check for `db` needed? Is this in case there is no config defined (from above)? If so, I'd say we throw there and drop the null check here.